### DISCUSSION TITLE
An error occurred when clicking the upscale button

### DISCUSSION THREAD

**Discussion by lushui2** [lllyasviel/stable-diffusion-webui-forge#1521] [General]
![1724688173990](https://github.com/user-attachments/assets/2c25c6fe-5124-43a2-8c7b-ada63a83040d)

**Comment by DenOfEquity** [comment]
This results from an oversight in a previous fix I made for the hrfix button: I didn't account for control images in the gallery. In the process of fixing it I've found more edge cases that can be resolved in a user-friendly way. Aiming to update soon.
